### PR TITLE
New package: Typst.Typst version 0.2.0

### LINKED ISSUE #102292: [Package Request]: Typst
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/typst/typst/releases
Publisher: Typst GmbH i.G.
Package Name: Typst
Description: A new markup-based typesetting system that is powerful and easy to learn.
Package Version: 0.2.0
Installer URL: https://github.com/typst/typst/releases/download/v0.2.0/typst-x86_64-pc-windows-msvc.zip

### PR DESCRIPTION
**Pull request by mdanish-kh**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

- Resolves #102292

Holding off merge until issue with RelativeFilePath not being added to env:PATH makes its way into a release.

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102958)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Comment by BrandonWanHuanSheng** [comment]
The pull request will not be affect because there is another version was released.

**Comment by BrandonWanHuanSheng** [comment]
I almost being tracked by the update server. You might can see when the publisher released the update. https://github.com/microsoft/winget-pkgs/pulls?q=is%3Apr+is%3Aopen+Typst.Typst
It will affect this link put the pull request won't affect.

**Comment by BrandonWanHuanSheng** [comment]
This PR is not merged so why I need to manually created folder directly for newer version. Also, this PR is waiting for new winget released.

**Comment by mdanish-kh** [comment]
@stephengillie Old PR here that was waiting on the stable release of WinGet 1.5; This needs to be re-validated please

File: manifests/t/Typst/Typst/0.2.0/Typst.Typst.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=NVS0.CRLF.7-3-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Typst.Typst
+PackageVersion: 0.2.0
+InstallerType: zip
+ReleaseDate: 2023-04-11
+Installers:
```
Comment (stephengillie) [inline manifests/t/Typst/Typst/0.2.0/Typst.Typst.installer.yaml:8]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Comment (stephengillie) [reply manifests/t/Typst/Typst/0.2.0/Typst.Typst.installer.yaml:8]: image

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**